### ISSUE TITLE
Write pipe error

### ISSUE THREAD

**Issue by albateh** [IBMPredictiveAnalytics/Model_Random_Forest#4]
Hi,

I've created Random Forest node for modeling. After this node, I used analysis node. When I run the analysis node, I get the following error:
"R_start.exe has stopped working. A problem caused the program to stop working correctly.  Please close the program." 
In the SPSS I am getting the error "Write pipe error".
Would appreciate any pointer to what is wrong that is causing the problem?

**Comment by hangtime79** [comment]
Make sure when you use the Analysis node setting you change to Field Name formatting.

![image](https://cloud.githubusercontent.com/assets/2651677/15546686/d5a4eb8a-226e-11e6-8c0e-7f71d9d42317.png)

**Comment by albateh** [comment]
Yes, I used Field name format option under Find Predicted/ Predictor fields using, If i didn't, it shows:  "No mating pairs of target and predictors fields are found', but the error "Write pipe error" happened...
